src/ripple/app/ledger/impl/LedgerCleaner.cpp

```diff
@@ -22,9 +22,8 @@
 #include <ripple/app/ledger/InboundLedgers.h>
 #include <ripple/app/ledger/LedgerMaster.h>
 #include <ripple/app/misc/LoadFeeTrack.h>
-#include <ripple/core/ThreadEntry.h>
 #include <ripple/protocol/JsonFields.h>
-#include <ripple/beast/core/Thread.h>
+#include <ripple/beast/core/CurrentThreadName.h>
```

Comment:
How strongly do you feel about this one?  I did a quick run at this and I'm getting compile errors out of MacOS-specific headers.  My suspicion is that the core.unity.cpp file is doing some favors for CurrentThreadName.cpp. I can get that straightened out, but it feels like there would probably be corresponding problems for Windows and Linux that would take me a while to get under control.